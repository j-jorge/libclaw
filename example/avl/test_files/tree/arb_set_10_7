 77 56 93 20 44 8 96 35 13 72 11