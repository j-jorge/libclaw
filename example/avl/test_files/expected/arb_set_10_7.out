Tree is empty
File loading...
size : 11
Min = 8
Max = 96
Tree's not empty
Found : 8 11 13 20 35 44 56 72 77 93 96
Scan with an iterator : 8 11 13 20 35 44 56 72 77 93 96
Scan bis, iterator affectation :8: 8 11 13 20 35 44 56 72 77 93 96
11: 11 13 20 35 44 56 72 77 93 96
13: 13 20 35 44 56 72 77 93 96
20: 20 35 44 56 72 77 93 96
35: 35 44 56 72 77 93 96
44: 44 56 72 77 93 96
56: 56 72 77 93 96
72: 72 77 93 96
77: 77 93 96
93: 93 96
96: 96
Deletions
Copy and scan
source size : 0
copy size : 0
Tree's copy is empty

Reverse scan

Assignment
Clear
Tree's is empty
